viz.yaml

```diff
@@ -18,21 +18,21 @@ info:
       version: 0.3.0
       name: USGS-VIZLAB/vizlab
       ref: NA
-    USAboundaries:
+    aws.s3:
+      repo: CRAN
+      version: 0.3.3
+    aws.signature:
       repo: CRAN
+      version: 0.3.5
+    USAboundaries:
+      repo: CRAN #lcarr had to manually install using devtools::install_github("ropensci/USAboundariesData")
```

Comment:
huh, i wonder why you had to go to github. `install.packages('USAboundaries')` works fine for me (just tried it again).

but also, we may not need to keep this package in here, after all, if the IPUMS dataset continues to be what we need. i guess let's hang onto it for a little while longer but try to remember to take it out soon if IPUMS keeps working.

Reply:
whoops - that comment was supposed to go next to USAboundariesData. USAboundaries installed from CRAN found but the data package would not.